testsuite/tests/win-unicode/test_env2.ml

```diff
@@ -0,0 +1,13 @@
+external set_environment_variable: string -> string -> unit = "caml_SetEnvironmentVariable"
+
+let print title = function
+  | None ->
+      Printf.printf "%s -> None\n%!" title
+  | Some s ->
+      Printf.printf "%s -> Some %S\n%!" title s
+
+let foo = "FOO"
+
+let () =
+  set_environment_variable foo "BAR";
```

Comment:
It's good to have this test (disabled), but perhaps indicates somehow that it is related to MPR#4499, not the current PR.

Also, it might be good to use non-ASCII characters in the string.